manifest.md

```diff
@@ -0,0 +1,261 @@
+<!--[metadata]>
++++
+draft = true
++++
+<![end-metadata]-->
+
+# Image Manifest
+
+The first goal is content-addressable images, by supporting an image model where the image's configuration can be hashed to generate an ID for the image and its components.
+The second goal is to allow multi-architecture images, through a "fat manifest" which references image manifests for platform-specific versions of an image.
+The third goal is to be translatable to the [OpenContainers/runtime-spec](https://github.com/opencontainers/runtime-spec)
+
+# Media Types
+
+The following media types are used by the manifest formats described here, and the resources they reference:
+
+- `application/vnd.oci.image.manifest.v1+json`: Image manifest format
+- `application/vnd.oci.image.manifest.list.v1+json`: Manifest list, aka "fat manifest"
+- `application/vnd.oci.image.serialization.v1+json`: Container config JSON
+- `application/vnd.oci.image.rootfs.tar.gzip`: "Layer", as a gzipped tar
+
+## Manifest List
+
+The manifest list is the "fat manifest" which points to specific image manifests for one or more platforms.
+Its use is optional, and relatively few images will use one of these manifests.
```

Comment:
Let's remove the commentary about "few images will use one of these".  Not really relevant for a spec, plus I'm not sure its true since I know of some folks who are working hard to port the popular images to other platforms.